app/views/declaration/summary/EoriOrAddress.scala

```diff
@@ -33,29 +34,34 @@ object EoriOrAddress {
     addressLabel: String,
     addressChangeLabel: String,
     changeController: Call,
+    mode: Mode,
```

Comment:
Why do we require the Mode here?
Just trying to understand the context.

Reply:
So that we can re-use this component helper to render the "ActionItem" (i.e. change links) or not